README.md

```diff
@@ -34,81 +35,96 @@ RAM is an app to create an appointment to test drive a motorcyle.
 </details>
 
 <details>
-  <summary>Server</summary>
+  <summary>Frontend</summary>
   <ul>
-    <li>Ruby-on-Rails</li>
+    <li>React</li>
   </ul>
 </details>
 
 <details>
-<summary>Database</summary>
+  <summary>Backend</summary>
   <ul>
-    <li>Postgress</li>
+    <li>Ruby on Rails</li>
   </ul>
 </details>
 
 ### Key Features <a name="key-features"></a>
 
 - Add appointment for a motorbike
 - Delete appointment for a motorbike
+- Create and Delete motorbikes
 
 
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
+<p align="right">(<a href="#table-of-contents">back to top</a>)</p>
 
 
-## 💻 Getting Started <a name="getting-started"></a>
+## Getting Started <a name="getting-started"></a>
 
 To get a local copy up and running, follow these steps.
 
-
 ### Prerequisites
 
-In order to run this project you need: any web-browser.
+In order to run this project you need: 
+- Any web-browser.
+- Latest version of Ruby on Rails
 
 ### Setup
 
 This API is consumed by its front-end counterpart [front-end](https://github.com/c00p75/appointment-app-frontend)
 
-
-Clone this [repo](https://github.com/c00p75/appointment-app-frontend.git) to your desired folder.
+Clone this [repo](https://github.com/c00p75/appointment-app-backend.git) to your desired folder.
 
 ```
-    git clone https://github.com/c00p75/appointment-app-frontend.git
+    git clone https://github.com/c00p75/appointment-app-backend.git
 ```
 
 ### Install
 
 Run from a terminal
 
 ```
-    cd appointment-app-frontend
+    cd appointment-app-backend
     bundle install
 ```
 
+### Database
+**To connect the project to the database:-**
+```
+    rails db:create
+    rails db:migrate
+```
+
 ### Usage
 
 **To run the project follow the following instruction:-**
 
 ```
-    rails db:migrate
-    rails s
+    rails server
 ```
 
-### Run tests
+### Run tests (using rspec)
 
-To run tests, using RSpec
+To run tests
 
 ```
-  rspec spec
+  bundle exec rspec
```

Comment:
- [ ]  You are using Rspec, but the "test" folder still exists, which is used by another test framework. Kindly consider deleting that folder if you are not using it. 👍

Reply:
fixed, thanks!